var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -393,7 +393,7 @@ class Trilinos(CMakePackage, CudaPackage, ROCmPackage):
         patch('xlf_tpetra.patch', when='@12.12.1 %' + _compiler)
     patch('fix_clang_errors_12_18_1.patch', when='@12.18.1%clang')
     patch('cray_secas_12_12_1.patch', when='@12.12.1%cce')
-    patch('cray_secas.patch', when='@12.14.1:%cce')
+    patch('cray_secas.patch', when='@12.14.1:12.99.99%cce')
```

Comment:
```suggestion
    patch('cray_secas.patch', when='@12.14.1:12%cce')
```
This is the newer cleaner way for spack ranges that extend to the end of a version

Reply:
Nice, I was looking for a better way to express that. Let me fix ...